<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferInquiryForm</class>
 <widget class="QWidget" name="TransferInquiryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地铁换乘查询</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="FPlabel">
         <property name="text">
          <string>终点站：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="SPlabel">
         <property name="text">
          <string>起点站：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="SPlineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>中南大学</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="FPlineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>铁道学院</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>换乘策略：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="transferradioButton">
       <property name="text">
        <string>少换乘</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="timeradioButton">
       <property name="text">
        <string>较快捷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inquirypushButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restorepushButton">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>换乘路线：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowserRoute"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
